fdbserver/workloads/RemoveServersSafely.actor.cpp

```diff
@@ -511,6 +511,45 @@ struct RemoveServersSafelyWorkload : TestWorkload {
 		return killProcArray;
 	}
 
+	// If a process is rebooted, it's processid will change. So we need to monitor
+	// such changes and re-issue the locality-based exclusion again.
+	ACTOR static Future<Void> checkLocalityChange(RemoveServersSafelyWorkload* self,
```

Comment:
I'm not to familiar with the test setup, but my understanding is that the excluded localities might differ between different runs, e.g. there could be a run where the `machineid` is used for exclusion and another run where the `processid` used (or some mixture of both). In theory it would be great if we could specify a static process identifier, I would guess in most real world use cases people will use a static identifier for the exclusion.

Reply:
Good point. In this test, it appears that the list of localities obtained from `getLocalitiesFromAddresses()` call is fixed.